.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: krew-index CI
+
+on:
+- push
+- pull_request
+
+jobs:
+  build:
+    name: Build and Tags
+    runs-on: ubuntu-latest
+    env: 
+      KREW_VERSION: v0.3.3
+      GO111MODULE: on
+    steps:
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Install kubectl
+      run: |
+        sudo curl -fsSL -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.13.1/bin/linux/amd64/kubectl
+        sudo chmod +x /usr/bin/kubectl
+
+    - name: Install krew
+      shell: bash
+      run: |
+        cd "$(mktemp -d)" &&
+        curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/download/${KREW_VERSION}/krew.{tar.gz,yaml}" &&
+        tar zxvf krew.tar.gz &&
+        ./krew-linux_amd64 install --manifest=krew.yaml --archive=krew.tar.gz
```

Comment:
@ahmetb I can't use pipefail with pipes in the last stage, pipefail does not work with pipes. But in this stage I can, just adding a shell label with bash, it contains pipefail.

Reply:
we'll need at least some `set -u`. I also must admit I don't fully understand your comment. What happens if you just add `set -euo pipefail;` to beginning of every script?